wagtail/admin/forms/choosers.py

```diff
@@ -44,6 +44,8 @@ class AnchorLinkChooserForm(forms.Form):
 class EmailLinkChooserForm(forms.Form):
     email_address = forms.EmailField(required=True)
     link_text = forms.CharField(required=False)
+    subject = forms.CharField(required=False)
+    body = forms.CharField(required=False)
```

Comment:
Can we please make this work as a multi-line field, something like...

```suggestion
    body = forms.CharField(required=False, widget=forms.Textarea(attrs={"rows": 3}))
```